src/test/java/uk/gov/di/ipv/service/ConfigurationServiceTest.java

```diff
@@ -53,18 +54,28 @@
     void setUp() throws URISyntaxException {
         credentialIssuers =
                 new CredentialIssuers(
-                        Set.of(
-                                new CredentialIssuerConfig(
-                                        "PassportIssuer",
-                                        new URI("http://www.example.com"),
-                                        new URI("http://www.example.com/credential")),
-                                new CredentialIssuerConfig(
-                                        "FraudIssuer",
-                                        new URI("http://www.example.com"),
-                                        new URI("http://www.example.com/credential"))),
+                        new HashSet<>(
+                                List.of(
+                                        new CredentialIssuerConfig(
+                                                "PassportIssuer",
+                                                new URI("http://www.example.com"),
+                                                new URI("http://www.example.com/credential")),
+                                        new CredentialIssuerConfig(
+                                                "FraudIssuer",
+                                                new URI("http://www.example.com"),
+                                                new URI("http://www.example.com/credential")))),
                         CREDENTIAL_ISSUER_CONFIG_BASE64_VERSION_1);
     }
 
+    @Test
+    void shouldReturnCredentialIssuersWhenPassedNull() {
+        ConfigurationService underTest = new ConfigurationService(ssmProvider);
```

Comment:
Should we move this out to a field now if several tests are using it?

Reply:
Sounds good. have just pushed the change.